cmake/FindClang.cmake

```diff
@@ -143,9 +143,10 @@ if(Clang_FOUND AND NOT TARGET Clang::Clang)
   set_target_properties(Clang::Clang PROPERTIES
     IMPORTED_LOCATION ${Clang_LIBRARY}
     INTERFACE_INCLUDE_DIRECTORIES "${Clang_INCLUDE_DIR};${Clang_BUILD_INCLUDE_DIR};${LLVM_INCLUDE_DIR};${LLVM_BUILD_INCLUDE_DIR}")
-
-  find_package(Curses REQUIRED)
-  find_package(ZLIB REQUIRED)
+  if(NOT WIN32)
```

Comment:
Does msys2 build need Curses & ZLIB ?

Reply:
I don't have msys2 environments. With VS2017 toolchain, these libs are not required.

Reply:
I've changed it to `if(NOT MSVC)`. I think msys2 llvm still uses it (but I don't have a Windows environment to test)